src/basic/env-util.c

```diff
@@ -274,6 +274,19 @@ _pure_ static bool env_match(const char *t, const char *pattern) {
         return false;
 }
 
+_pure_ static bool env_entry_has_name(const char *entry, const char *name) {
```

Comment:
_pure_ is kinda unnecessary if the function is static anyway, no? the compiler can figure this out on its own really..